cmd/probe/cli/run.go

```diff
@@ -0,0 +1,115 @@
+/*
+Copyright (C) 2022-2023 ApeCloud Co., Ltd
+
+This file is part of KubeBlocks project
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+package cli
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/signal"
+	"strings"
+	"syscall"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	configFile       string
+	port             int
+	grpcPort         int
+	internalGRPCPort int
+	logLevel         string
+	componentsPath   string
+	enableAppHealth  bool
+)
+
+const (
+	runtimeWaitTimeoutInSeconds = 60
+)
+
+var RunCmd = &cobra.Command{
+	Use:   "run",
+	Short: "Run sqlchannel and db service.",
+	Example: `
+sqlctl run  -- mysqld
+  `,
```

Comment:
mysqld is an arg to `sqlctl run`? and cobra.ExactArgs should be 1?

Reply:
no, it can have more than one args